### PR TITLE
feat(rest): add cli and docker image publishing

### PR DESCRIPTION
**Pull request by TimoGlastra**
Add CLI to the rest (makes it usable without code) and also adds docker publishing. This means we don't have to set up a boilberplate project every time we use the rest server.

### CODE REVIEW

**Review by janrtvld — APPROVED** [review]

File: packages/rest/README.md [hunk 1]
```diff
@@ -95,3 +95,34 @@ In case of an event, we will send the event to the webhookUrl with the topic of
...
+      # possible to set values using env variables
+      AFJ_REST_LOG_LEVEL: 1
+    volumes:
+      # also possible to set values using json
+      - ./myConfig.json:/config.json
+    ports:
+      - '5000:5000'
+      - '5001:5001'
+      - '3000:3000'
+    platform: linux/amd64
```
Comment (blu3beri) [inline packages/rest/README.md:119]: Will this work on m1?

File: packages/rest/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04 as base
```
Comment (blu3beri) [inline packages/rest/Dockerfile:1]: I think you wanted me to do this once and I kinda forgot, but don't we want to create an afj image with libindy and all installed?

**Review by blu3beri — APPROVED** [review]
I really like the addition! I do think that it could benefit from some documentation. Like a README and some minor in-code documentation.

File: packages/rest/README.md (hunk 1 above)
Comment (TimoGlastra) [reply packages/rest/README.md:119]: Partly. It works until you try to open a connection to the ledger :(

Libindy is not available for ubuntu arm. However, BCGov is working on ARM images that include libindy I think.

File: packages/rest/Dockerfile (hunk 2 above)
Comment (TimoGlastra) [reply packages/rest/Dockerfile:1]: Yeah may be still worth looking at.

(here's the issue: https://github.com/animo/indy-nodejs-docker/issues/1)

**Comment by TimoGlastra** [comment]
> I do think that it could benefit from some documentation. Like a README and some minor in-code documentation.

Sure will take add some documentation!

**Pull request merged**